prototype/pca_compression/compress.py

```diff
@@ -0,0 +1,75 @@
+import serial
```

Comment:
Is it ok for us to be dependant on pySerial

Reply:
The prototype that was already present in the library (mnist) depends on tensorflow, panda and sklearn. Therefore, I assumed it was alright to depend on it, but we could give a requirements.txt in the prototype folder to allow users to directly run every prototypes.